src/sample/lecDashController.java

```diff
@@ -233,7 +234,7 @@ public void validateNRIC(){
       } else{
         Message.setText("");
         saveButton.setDisable(false);
-        name.setDisable(true);
+        name.setDisable(false);
```

Comment:
why false?

Reply:
Because the nric got no error le, so they can edit their name

Reply:
If nric got error, they cannot edit name